### PR TITLE
Exo createscenariopage

### PR DESCRIPTION
**Pull request by gremir10**

### CODE REVIEW

**Comment by cookjackc** [comment]
Tried a fresh install, ran into a few issues.
The following items are missing from requirements/prod.txt and so are not gathered by the install script:

#JWT
jwt==1.3.1
PyJWT==1.7.1
Flask-JWT-Simple==0.0.3

# DB-Schemata
marshmallow==3.20.1
marshmallow-sqlalchemy==0.29.0
flask-marshmallow==0.15.0

Additionally, the new install script isn't being used, so the axios config isn't being automatically updated and that side of the app is not testable.

Two issues in the legacy app: 
The navbar still covers the "Create Scenario" button and name field - hopefully easy styling fix by adding some margins.

Line 401 of user/views.py:
https://github.com/edurange/edurange-flask/blob/f43f0e9f6fa9bccf9c93737bccdcb79f0724cfc2/edurange_refactored/user/views.py#L401

What is `namedict` here? It doesn't appear to be defined anywhere.

I'll try another fresh install later and test some more to verify.

**Comment by cookjackc** [comment]
Remediation step 1: 

I have a version of this branch installed on google cloud, which is synchronized with staging such that the requirements files and install script are correct.

It appears to be working, up to the point that the `namedict` variable is causing problems. It is not defined anywhere, and as far as I can tell - is not used, where did this come from?

This is causing problems in 
https://github.com/edurange/edurange-flask/blob/f43f0e9f6fa9bccf9c93737bccdcb79f0724cfc2/edurange_refactored/tasks.py#L94

Additionally, some query appears to be returning a "Row" object instead of a serializable "dict", and that breaks scenario creation. Working on finding that particular item. 

Partial Traceback:
```
[FLASK]   File "/usr/lib/python3.10/json/encoder.py", line 199, in encode
[FLASK]     chunks = self.iterencode(o, _one_shot=True)
[FLASK]   File "/usr/lib/python3.10/json/encoder.py", line 257, in iterencode
[FLASK]     return _iterencode(o, 0)
[FLASK]   File "/home/edurange_flask/.local/lib/python3.10/site-packages/kombu/utils/json.py", line 45, in default
[FLASK]     return super().default(o)
[FLASK]   File "/usr/lib/python3.10/json/encoder.py", line 179, in default
[FLASK]     raise TypeError(f'Object of type {o.__class__.__name__} '
[FLASK] kombu.exceptions.EncodeError: Object of type Row is not JSON serializable
```

File: edurange_refactored/flask/modules/routes/student_routes.py [hunk 1]
```diff
@@ -84,6 +102,94 @@ def jwt_test():
...
+
+        # s_id, g_id = s_id._asdict(), g_id._asdict()
+
+        gid = g_id["id"]
+        student_ids = db_ses\
+                    .query(GroupUsers.id)\
+                    .filter(GroupUsers.group_id == gid)\
+                    .all()
+
+        namedict = gen_chat_names(student_ids, s_id_list)
```
Comment (WinstonShine) [inline edurange_refactored/flask/modules/routes/student_routes.py:177]: Does this commented out code plan to be used later? I also see an assignment to a variable called namedict inside the commented out code, not sure if its related to what Jack mentioned.

**Comment by exoriparian** [comment]
Scenario Creation is now handled in /instructor_routes.py via scenario_interface ; deprecated

**Pull request closed without merging**